vispy/util/wrappers.py

```diff
@@ -40,10 +40,8 @@ def use(app=None, gl=None):
             * 'SDL2': use SDL v2 backend.
             * 'osmesa': Use OSMesa backend
         Additional backends:
-            * 'ipynb_webgl': run vispy from a Jupyter notebook (not fully
-               functional)
-            * 'ipynb_vnc': render in a Jupyter notebook via a VNC approach
-              (experimental)
+            * 'jupyter_rfb': show vispy canvases in Jupyter lab/notebook.
```

Comment:
Do you maybe want to mention that this is "via 'jupyter_rfb'" or something about it depending on that package?

Reply:
done!